uber/templates/registration/index_base.html

```diff
@@ -10,30 +10,34 @@
     });
 
     $(function(){
+        function decrypt_checkin_code(barcode) {
```

Comment:
By putting this function inside a document-is-ready handler, it's not going to be globally available, but I suspect that you want it to be.  If so then you should move the ``function decrypt_checkin_code`` out of the ``$(function () {`` block so that it's declared globally.